src/css/typography.sass

```diff
@@ -0,0 +1,165 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins:wght@400;500;600;700&display=swap")
+@import 'colors'
+
+// $thin: 100
+// $extra-light: 200
+// $light: 300
+$regular: 400
+$medium: 500
+$semi-bold: 600
+$bold: 700
+// $extra-bold: 800
+// $black: 900
+// $extra-black: 950
+
+=header-globals()
+  font-family: 'Poppins', sans-serif
```

Comment:
Set the font family once to be in line with the design spec

Reply:
Done